Zone5/Transport/Zone5HTTPClient.swift

```diff
@@ -83,7 +83,7 @@ final public class Zone5HTTPClient {
 	///   - block: The function containing the work to perform. It receives a strong copy of the parent `Zone5` class, and the configured `baseURL`.
 	private func execute<T>(with completion: @escaping (_ result: Result<T, Zone5.Error>) -> Void, _ block: (_ zone5: Zone5) throws -> PendingRequest) -> PendingRequest? {
 		do {
-			guard let zone5 = zone5, zone5.isConfigured else {
+			guard let zone5 = zone5, zone5.baseURL != nil else {
```

Comment:
This should probably still be calling `isConfigured` which checks for a `clientID` or `accessToken` (as requests will presumably fail without one or the other?).

Reply:
They are supposed to fail with a server error, not a client side error.

Reply:
👍🏼 can we document that with a comment then, please?

Reply:
Sure. I wrote it in the description of this PR. I'll add it to a few places in the code.

Reply:
done